crates/sequencer/src/sequencer.rs

```diff
@@ -556,10 +561,10 @@ where
 
     /// Signs batch of messages with sovereign priv key turns them into a sov blob
     /// Returns a single sovereign transaction made up of multiple ethereum transactions
-    fn make_blob(&mut self, raw_message: Vec<u8>) -> Vec<u8> {
+    fn make_blob(&mut self, raw_message: Vec<u8>, storage: &Sm::NativeStorage) -> Vec<u8> {
```

Comment:
why are we passing down storage instead of using self.storage?

Reply:
Because there is no more self.storage